src/vs/workbench/contrib/chat/browser/chatContentParts/chatAttachmentsContentPart.ts

```diff
@@ -141,6 +141,8 @@ export class ChatAttachmentsContentPart extends Disposable {
 			widget = this.instantiationService.createInstance(PromptTextAttachmentWidget, attachment, undefined, { shouldFocusClearButton: false, supportsDeletion: false }, container, this._contextResourceLabels);
 		} else if (resource && (attachment.kind === 'file' || attachment.kind === 'directory')) {
 			widget = this.instantiationService.createInstance(FileAttachmentWidget, resource, range, attachment, correspondingContentReference, undefined, { shouldFocusClearButton: false, supportsDeletion: false }, container, this._contextResourceLabels);
+		} else if (isTerminalCommandVariableEntry(attachment)) {
+			widget = this.instantiationService.createInstance(TerminalCommandAttachmentWidget, attachment, undefined, { shouldFocusClearButton: false, supportsDeletion: false }, container, this._contextResourceLabels);
```

Comment:
So this is rendered differently when you attach the terminal tool? Or what is a `ITerminalCommandVariableEntry`?